src/internal/components/drawer/styles.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+/* stylelint-disable @cloudscape-design/no-implicit-descendant, selector-max-type */
+@use '../../styles/tokens' as awsui;
+@use '../../styles' as styles;
+
+.drawer {
+  @include styles.styles-reset;
+  word-wrap: break-word;
+  padding: awsui.$space-scaled-l awsui.$space-panel-side-right awsui.$space-scaled-xxxl awsui.$space-panel-side-left;
+}
+
+.header {
+  @include styles.font-panel-header;
```

Comment:
Would it make sense to de-duplicate this block?

Reply:
Do you mean make it the same for help panel and drawer? The header scss is already different, since the drawer header doesn't have the same style overrides as help panel. And I don't want to overlap them too much, since they may diverge even more in the future (ie, there are talks of putting other action buttons in the drawer header).

Reply:
The CSS rules in this `.header` selector look exactly the same to me as in help panel (see [here](https://github.com/cloudscape-design/components/blob/main/src/help-panel/styles.scss#L107)) with the only exception that the rule `border: 0` is missing here in the drawer —I assume you removed it because it's actually unnecessary (and it seems unnecessary in the help panel too).

Not sure how much if at all adding action buttons in the drawer header would need to affect this block, with this information I would personally still be inclined to deduplicate, but won't block the PR on this.